cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -239,6 +245,93 @@ struct null_considering_binop {
   }
 };
 
+rmm::device_uvector<order> get_orders(binary_operator op,
+                                      uint32_t const num_columns,
+                                      rmm::cuda_stream_view stream)
+{
+  std::vector<order> op_modifier(
+    num_columns,
+    (op == binary_operator::LESS || op == binary_operator::GREATER_EQUAL) ? order::ASCENDING
```

Comment:
I'm a bit confused. Why "<" and ">=" ops are treated as the same?

Reply:
`A >= B`  is equivalent to `!(A < B)`